NotificationHubs/src/com/windowsazure/messaging/BrowserRegistration.java

```diff
@@ -0,0 +1,119 @@
+//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//----------------------------------------------------------------
+
+package com.windowsazure.messaging;
+
+import java.util.Objects;
+
+/**
+ * This class represents a Browser Push device registration.
+ */
+public class BrowserRegistration extends Registration {
+    private static final String BROWSER_NATIVE_REGISTRATION1 = "<?xml version=\"1.0\" encoding=\"utf-8\"?><entry xmlns=\"http://www.w3.org/2005/Atom\"><content type=\"application/xml\"><BrowserRegistrationDescription xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://schemas.microsoft.com/netservices/2010/10/servicebus/connect\">";
+    private static final String BROWSER_NATIVE_REGISTRATION2 = "<Endpoint>";
```

Comment:
I saw these names above too, so I see you're being consistent with the existing code base, but it seems incredibly undescriptive to me. Could we add names like "XML_HEADER", "OPEN_ENDPOINT_BLOCK", etc, or something that alludes to the purpose of it a little more?

Reply:
Agreed, it was following convention as to not rock the boat in terms of consistency from the other classes without renaming them as well.  I don't mind the numbering because at least you know the proper order of things instead of having to guess which is next.